sunpy/cm/color_tables.py

```diff
@@ -101,177 +98,193 @@ def aia_color_table(wavelength):
-    118, 119, 121, 122, 123, 124, 126, 127, 128, 130, 131, 132, 133, 135, 136,
-    137, 138, 140, 141, 142, 144, 145, 146, 147, 149, 150, 151, 153, 154, 155,
-    156, 158, 159, 160, 161, 163, 164, 165, 167, 168, 169, 170, 172, 173, 174,
-    175, 177, 178, 179, 181, 182, 183, 184, 186, 187, 188, 189, 191, 192, 193,
-    195, 196, 197, 198, 200, 201, 202, 204, 205, 206, 207, 209, 210, 211, 212,
-    214, 215, 216, 218, 219, 220, 221, 223, 224, 225, 226, 228, 229, 230, 232,
-    233, 234, 235, 237, 238, 239, 240, 242, 243, 244, 246, 247, 248, 249, 251,
-    252, 253, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
-    255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
-    255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
-    255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255
-])
-
-eit_yellow_g = np.array([
-    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
-    21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39,
-    41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59,
-    60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78,
-    79, 80, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98,
-    99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113,
-    114, 115, 116, 117, 118, 119, 120, 121, 123, 124, 125, 126, 127, 128, 129,
-    130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144,
-    145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159,
-    160, 161, 162, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175,
-    176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190,
-    191, 192, 193, 194, 195, 196, 197, 198, 199, 200, 201, 202, 203, 205, 206,
-    206, 207, 209, 209, 210, 210, 212, 213, 213, 215, 216, 216, 218, 219, 219,
-    221, 221, 222, 224, 224, 225, 227, 227, 228, 230, 230, 231, 231, 232, 234,
-    234, 235, 237, 237, 238, 240, 240, 241, 241, 243, 244, 244, 246, 247, 247,
-    249, 250, 250, 252, 252, 253, 255, 255, 255
-])
-
-eit_yellow_b = np.concatenate(
-    (np.zeros(201).astype('int'),
-     np.array([
-         7, 7, 15, 22, 22, 30, 30, 37, 45, 45, 52, 60, 60, 67, 75, 75, 82, 82,
-         90, 97, 97, 105, 112, 112, 120, 127, 127, 135, 135, 142, 150, 150,
-         157, 165, 165, 172, 180, 180, 187, 187, 195, 202, 202, 210, 217, 217,
-         225, 232, 232, 240, 240, 247, 255, 255, 255
-     ])))
-
-eit_dark_blue_r = np.concatenate(
-    (np.zeros(206).astype('int'),
-     np.array([
-         9, 13, 21, 25, 25, 29, 33, 41, 49, 53, 57, 65, 69, 73, 77, 77, 85, 94,
-         98, 102, 110, 114, 118, 122, 134, 134, 138, 142, 146, 154, 158, 162,
-         166, 179, 183, 183, 187, 191, 199, 203, 207, 215, 223, 227, 231, 231,
-         235, 243, 247, 255
-     ])))
-
-eit_dark_blue_g = np.concatenate(
```

Comment:
why does it look like you moved all of this? Is it just a bad diff or did you have a reason to move color tables around?